ajax/libs/angular-growl/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "angular-growl",
+  "version": "0.4.0",
+  "description": "growl like notifications for angularJS projects, using bootstrap alert classes",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/marcorinck/angular-growl.git"
+  },
+  "author": {
+    "name": "Marco Rinck"
+  },
+  "license": "MIT",
+  "filename": "angular-growl.min.js",
```

Comment:
Let's move this line between `name` and `version` fields.
It's more useful for developers.

Thanks.